assets/js/components/Collection/Search.jsx

```diff
@@ -1,19 +1,25 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Link } from "react-router-dom";
 
-const CollectionSearch = () => {
-  let items = [];
+const CollectionSearch = ({ collection }) => {
+  const [filteredWorks, setFilteredWorks] = useState(collection.works);
```

Comment:
Can you double check that `collection.works` defaults to an empty array if no works are added yet to a collection?  Otherwise we need to wrap this code in a check so things don't crash.

Reply:
Yes, `collection.works` will always return an array, as long as it is included as part of the graphQL query